services/common/src/redux/customAxios.js

```diff
@@ -34,10 +62,33 @@ const CustomAxios = ({ errorToastMessage, suppressErrorNotification = false } =
         (errorToastMessage === "default" || errorToastMessage === undefined) &&
         !suppressErrorNotification
       ) {
-        notification.error({
-          message: formatErrorMessage(error?.response?.data?.message ?? String.ERROR),
-          duration: 10,
-        });
+        console.error('Detailed Error: ', error?.response?.data?.detailed_error)
```

Comment:
Missed a console log 😜

Reply:
Thanks @matbusby-fw,
Actually, I kept this purposely, as the detailed error will not be visible to FE anymore with this change. But it may not good practice. Shall I remove it?